### DISCUSSION TITLE
Knot resolver

### DISCUSSION THREAD

**Discussion by Delkaloop** [trinib/Adguard-Wireguard-Unbound-Cloudflare#34] [Ideas]
What do you think about Knout Resolver? Looks like its much better than Unbound.

https://github.com/CZ-NIC/knot-resolver

**Comment by trinib** [comment]
To say which is better between Knot and Unbound security is not my position to say,  the technical gibberish from it's documents is only  for seasoned professional network devs to understand and compare lol. And actually comparison test for these DNS resolvers from google is mostly the results from how many use it and recommend/share it.

Both are them are good from what I read.  [Unbound](https://github.com/NLnetLabs/unbound) seems to be the more mature project with the repo 14 years&1.7k stars and [Knot](https://github.com/CZ-NIC/knot-resolver) that is 8 years&244 stars .

**Reply by trinib** [reply]
@Delkaloop  created [poll](https://github.com/trinib/AdGuard-WireGuard-Unbound-Cloudflare/discussions/35#discussion-4079676)

**Comment by trinib** [comment]
> What do you think about Knout Resolver? Looks like its much better than Unbound.
> 
> https://github.com/CZ-NIC/knot-resolver

Knot seems to be more flexible than Unbound like DNScrypt compare to stubby.. I guess I'll add to guide eventually. Still cannot say which has better security protocols.
